com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs

```diff
@@ -899,6 +896,10 @@ void InitializeLightLoop(IBLFilterBSDF[] iBLFilterBSDFArray)
                 Shader.EnableKeyword("SCREEN_SPACE_SHADOWS_OFF");
             }
 
+            if (ShaderConfig.s_EnableProbeVolumes == 1)
+            {
+                ProbeReferenceVolume.instance.InitProbeReferenceVolume(1024, m_Asset.currentPlatformRenderPipelineSettings.probeVolumeMemoryBudget, new Vector3Int(1024, 64, 1024));
```

Comment:
what are the hardcoded value 1024 and other? (can be fix in next pr)

Reply:
The first 1024 is just how much the pool allocator uses as size for the brick pool. It is a value that we need to tweak and see on profiling if it makes a difference changing it, realistically we just don't need it as a parameter of the Init function, but would need investigation whether it makes sense for different pipelines (not different projects) to  have different values. 

Note that the value is not really that hardcoded, as in it is a fixed number in a constructor, but then the value itself is correctly named in the rest of the code. 


The second set is again just whatever is the default for the index dimension of the reference volume. It will be overwritten at first load if it mismatches what's on the profile. We can give it a name and store as static somewhere indeed.